app/views/simplified-account/settings/stripe-details/responsible-person/check-your-answers.njk

```diff
@@ -22,7 +22,7 @@
           {
             href: changeResponsiblePersonLink,
             text: "Change",
-            visuallyHiddenText: "change name",
+            visuallyHiddenText: "name",
```

Comment:
Similar to the above "Edit" comment

Reply:
As above - this will read as "Change change name" on a screen reader.